9c-internal/common-dbs/application.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: common-dbs
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/planetarium/9c-infra.git
+    targetRevision: new-arena-service
```

Comment:
not changed yet

Reply:
I was planning to work on this branch while proceeding with 200280.  
If we merge it into `main`, wouldn't the continuous sync notifications be inconvenient?